google/cloud/security/inventory/inventory_loader.py

```diff
@@ -250,25 +248,22 @@ def main(argv):
     crm_rate_limiter = RateLimiter(max_crm_calls, 100)
 
     pipelines = [
-        {'pipeline': load_projects_pipeline,
-         'status': ''},
-        {'pipeline': load_projects_iam_policies_pipeline,
-         'status': ''},
-        {'pipeline': load_org_iam_policies_pipeline,
-         'status': ''},
+        LoadOrgIamPoliciesPipeline(
+            cycle_timestamp, configs, crm_rate_limiter, dao),
+        # TODO: add load projects pipeline
+        # TODO: add load project policies pipeline
     ]
 
+    succeeded = []
     for pipeline in pipelines:
```

Comment:
There is a lot happening in main() which makes testing hard. It would be worth just doing the bare initialization required in main then calling out to one or more functions that build, run and report the pipeline status, which can be tested. (Note for a future PR)